pdns/libssl.cc

```diff
@@ -806,6 +806,11 @@ std::unique_ptr<SSL_CTX, void(*)(SSL_CTX*)> libssl_init_server_context(const TLS
 {
   auto ctx = std::unique_ptr<SSL_CTX, void(*)(SSL_CTX*)>(SSL_CTX_new(SSLv23_server_method()), SSL_CTX_free);
 
+  if (!ctx) {
+    ERR_print_errors_fp(stderr);
+    throw std::runtime_error("Error creating an OpenSSL server context");
```

Comment:
Is it important to have the errors printed here? If not, then why not use `throw pdns::OpenSSL::error(...)`?

Reply:
That would be better indeed, I just have to get used to it :)